README.md

```diff
@@ -89,7 +89,7 @@ The following tables show the values to use in the `mbed_app.json` file for your
 
 ##### Thread commissioning
 
-By default, the Thread application uses the static network link configuration defined in the [mesh API configuration file](https://github.com/ARMmbed/mbed-os/blob/master/features/nanostack/FEATURE_NANOSTACK/mbed-mesh-api/mbed_lib.json).
+By default, the Thread application uses the static network link configuration defined in the [mesh API configuration file](https://github.com/ARMmbed/mbed-os/blob/master/features/nanostack/mbed-mesh-api/mbed_lib.json).
```

Comment:
This needs to be changed once 5.9 docs are published because now Nanostack/Mesh configuration page is up to date.
https://os-doc-builder.test.mbed.com/docs/development/reference/configuration-mesh.html